web/packages/teleport/src/ironrdp/src/lib.rs

```diff
@@ -162,7 +162,7 @@ impl FastPathProcessor {
                 user_channel_id,
                 // These should be set to the same values as they're set to in the
                 // `Config` object in lib/srv/desktop/rdp/rdpclient/src/client.rs.
-                no_server_pointer: true,
+                no_server_pointer: false,
                 pointer_software_rendering: false,
```

Comment:
`pointer_software_rendering: true` means IronRDP will render cursor by updating screen (so on every mouse move it will return graphics update), we don't do that because browser will render cursor for us

Reply:
Perfect, that's what I'd expect then.